buildenv/jenkins/docker-slaves/README.md

```diff
@@ -0,0 +1,43 @@
+# Docker
+## About
+Dockerfiles are used to define how a docker container/image
+should be built. Docker containers are essentially virtual
+machines designed for short term use. This makes them very
+useful for testing purposes as you can generate a new container
+for every test, so you will know you are running against the
+same environment every time.
+## Platforms currently available:
```

Comment:
My research tells me Mac in a container is not possible without a somewhat hacky solution, but Windows on Windows is possible.